### PR TITLE
Fix NavigationBar.Toggle button so that it toggles collapse of Naviga…

### LINKED ISSUE #83: Bootstrap Collapse button not working
The NavigationBar.Toogle() button does not actually open the menu when clicked. This seems to be caused by missing HTML properties:

  data-toggle="collapse" data-target="#myNavbar">

### PR DESCRIPTION
**Pull request by mkotsbak**
…tionBar.Collapse.

Closes #83

### CODE REVIEW

File: js/src/main/scala/org/widok/bindings/Bootstrap/package.scala [hunk 1]
```diff
@@ -287,6 +289,7 @@ package object Bootstrap {
         HTML.Container.Inline().css("icon-bar")
       ).css("navbar-toggle", "collapsed")
         .attribute("type", "button")
+        .onClick(_ => collapseClosed := !collapseClosed.get)
```
Comment (tindzk) [inline js/src/main/scala/org/widok/bindings/Bootstrap/package.scala]: You can shorten this to ``collapseClosed.update(!_)``.

**Comment by tindzk** [comment]
Looks good. The only limitation I see is that you introduce a global variable. I would prefer if the user had to explicitly pass it.

**Comment by mkotsbak** [comment]
Fixed update().

**Comment by mkotsbak** [comment]
To avoid the global variable, I think Toggle and Collapse must be classes with methods to set the state Var. Hmm, or, since this was not working, we could probably break the API and make it:
Toggle(state: WriteChannel[Boolean])(widgets*)
Collapse(state: ReadChannel[]

**Comment by mkotsbak** [comment]
Created a new pull request with that solution.

**Comment by tindzk** [comment]
The problem with the global state is that a user could instantiate a ``Toggle()``/``Collapse()`` widget only once, otherwise the state would interfere. This case may be a bit contrived, but you never know how people are going to use your library.
